api/src/pcapi/routes/backoffice/finance/finance_incidents_blueprint.py

```diff
@@ -441,18 +601,47 @@ def create_collective_booking_incident(collective_booking_id: int) -> utils.Back
         flash(utils.build_form_error_msg(form), "warning")
         return redirect(request.referrer, 303)
 
-    booking_has_no_incident = validation.check_incident_collective_booking(collective_booking)
-    is_valid_amount = validation.check_total_amount(
-        input_amount=collective_booking.total_amount,
-        bookings=[collective_booking],
+    if not (valid := validation.check_incident_collective_booking(collective_booking)):
+        for message in valid.messages:
+            flash(message, "warning")
+        return redirect(redirect_url, 303)
+
+    incident = finance_api.create_overpayment_finance_incident_collective_booking(
+        collective_booking, origin=form.origin.data
+    )
+    incident_url = url_for(
+        "backoffice_web.finance_incidents.get_incident",
+        finance_incident_id=incident.id,
     )
 
-    if not (valid := (booking_has_no_incident and is_valid_amount)):
+    flash(Markup('Un nouvel <a href="{url}">incident</a> a été créé.').format(url=incident_url), "success")
+    return redirect(redirect_url, 303)
+
+
+@finance_incidents_blueprint.route(
+    "/collective-bookings/<int:collective_booking_id>/create-commercial-gesture", methods=["POST"]
+)
+@utils.permission_required(perm_models.Permissions.MANAGE_INCIDENTS)
+def create_collective_booking_commercial_gesture(collective_booking_id: int) -> utils.BackofficeResponse:
+    collective_booking = educational_models.CollectiveBooking.query.filter_by(id=collective_booking_id).one_or_none()
+    if not collective_booking:
+        raise NotFound()
+
+    redirect_url = request.referrer or url_for("backoffice_web.collective_bookings.list_collective_bookings")
+    form = forms.CollectiveCommercialGestureCreationForm()
+
+    if not form.validate():
+        flash(utils.build_form_error_msg(form), "warning")
+        return redirect(request.referrer, 303)
```

Comment:
Je pense que tu voulais : 
```suggestion
        return redirect(redirect_url, 303)
```